calc/__init__.py

```diff
@@ -1,3 +1,5 @@
 __author__ = 'Reilly Tucker Siemens, Alex LordThorsen'
 __email__ = 'reilly@tuckersiemens.com, alexlordthorsen@gmail.com'
 __version__ = '0.1.0'
+
+from .calc import INTEGER, EOF, PLUS, Calc, CalcError
```

Comment:
Alternatively, what if I just described _why_ we do that right there in the code?